<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QE_TWin_Scale_Select_Dialog</class>
 <widget class="QWidget" name="QE_TWin_Scale_Select_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>200</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Window Title</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="horizontalSpacing">
    <number>4</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>17</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>168</width>
       <height>17</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Maximum</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="scaleTwoMaxEdit">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>168</width>
       <height>23</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="scaleTwoNameLabel">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>17</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>17</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="scaleTwoMinEdit">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>168</width>
       <height>23</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="scaleOneNameLabel">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>17</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>17</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="scaleOneMinEdit">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>168</width>
       <height>23</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="scaleOneMaxEdit">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>168</width>
       <height>23</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>27</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>168</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>17</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>168</width>
       <height>17</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Minumun</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scaleOneMinEdit</tabstop>
  <tabstop>scaleOneMaxEdit</tabstop>
  <tabstop>scaleTwoMinEdit</tabstop>
  <tabstop>scaleTwoMaxEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
